src/riak_core_info_service_process.erl

```diff
@@ -0,0 +1,113 @@
+-module(riak_core_info_service_process).
```

Comment:
The original "API" was leveldb sending custom Erlang messages that would not be routed via OTP. I felt strongly that implementing an OTP behavior would be effectively lying about the role the process played.

I haven't looked at the new code to see what's been done to it.